Makefile

```diff
@@ -336,6 +336,22 @@ test-e2e-daemon:
 	  --daemon-context-subject-prefix="$(OUTDIR)/lima/bin/limactl shell finch sudo" \
 	  --daemon-context-subject-env="LIMA_HOME=$(OUTDIR)/lima/data"
 
+# TODO: Blkio devices are not getting created in test environment, skipping for now
+.PHONY: test-e2e-daemon-linux
+test-e2e-daemon-linux:
+	# Create symlink for buildkit socket path compatibility
+	mkdir -p /run/buildkit
+	mkdir -p /var/lib/finch/buildkit
+	ln -sf /var/lib/finch/buildkit/buildkitd.sock /run/buildkit/buildkitd.sock || true
```

Comment:
Shouldn't setting `BUILDKIT_HOST` as we do in the service file make this unnecessary?

Reply:
You're correct, I forgot that the latest finch-daemon doesn't have the newest nerdctl version at the moment